pyracf/scripts/setup_precheck.py

```diff
@@ -0,0 +1,35 @@
+from pyracf import ResourceAdmin, SecurityRequestError
+
+
+def setup_precheck():
+    resource_admin = ResourceAdmin()
+    try:
+        access = resource_admin.get_my_access("IRR.IRRSMO00.PRECHECK", "XFACILIT")
+    except SecurityRequestError:
+        traits_precheck = {"base:universal_access": "None"}
+        result = resource_admin.add(
+            "IRR.IRRSMO00.PRECHECK", "XFACILIT", traits=traits_precheck
+        )
```

Comment:
If the XFACILIT class is RACLISTED then yes, otherwise no. I will change the script to do that where necessary. I'll also add unit testing that accounts for authority that the user may or may not have to actually do that.

Reply:
Oh I did actually say "You may also need to refresh the XFACILIT class" in the message printed out if the class gets added. I could make it do this programmatically, but there could be some hiccups with user authority that it could trip over, and I wanted to keep this simple since anyone can theoretically run this. I can change it though, if you think that would be best.

Reply:
Right, I see that in the message. Given that the setropts refresh could cause issues in some cases, I think this is probably good enough, especially because this nuance is mentioned in the message.